pytype/tools/merge_pyi/merge_pyi.py

```diff
@@ -626,6 +626,12 @@ def insert_comment_annotation(self, src_sig, pyi_sig):
       return []
 
   def can_annotate(self, src_sig):
+    if src_sig.short_name.startswith('__') and \
+            src_sig.short_name.endswith('__') and \
+            src_sig.short_name != "__init__":
```

Comment:
In addition to `__init__`, please also check for `__new__`.

Reply:
Done. But I'm not much familiar with this codebase. Can you give me a reference to read more about this change?

Reply:
Do you mean a reference for `__new__`? It runs before `__init__` to create an instance of a class: https://docs.python.org/3/reference/datamodel.html#object.__new__.

Reply:
Oh, that way. Got it. Thanks!